temporal/api/cloud/nexus/v1/message.proto

```diff
@@ -0,0 +1,114 @@
+syntax = "proto3";
+
+package temporal.api.cloud.nexus.v1;
+
+option go_package = "go.temporal.io/api/cloud/nexus/v1;nexus";
+
+import "google/protobuf/timestamp.proto";
+
+message IncomingServiceSpec {
+    // The name of the incoming service. Must be unique within an account
+    // The name must match `[a-zA-Z_][a-zA-Z0-9_]*`.
+    string name = 1;
+
+    // The namespace to route requests to
+    string namespace = 2;
+
+    // The task queue to route requests to
+    string task_queue = 3;
+
+    // The set of authorization policies for the service. Each request's caller
+    // must match with at least one of the specs to be accepted by the incoming service
+    repeated IncomingServiceAuthorizationSpec authorization_spec = 4;
```

Comment:
Does the ordering of these specs mean anything? If yes, we need to make sure we can retrieve the same ordering that the user provides.

Reply:
i think the ordering doesn't matter here (minus any form of short-circuit optimization), but cc @prasek @bergundy if you have thoughts